### PR TITLE
add acronyms to enum.py

### PR DESCRIPTION
**Pull request by InvisibleSymbol**
(sorry this took so long, git was kinda annoying...)

### CODE REVIEW

File: circleguard/enums.py [hunk 1]
```diff
@@ -11,34 +11,34 @@ class Error(Enum):
...
-    FadeIn         = 1048576
-    Random         = 2097152
-    LastMod        = 4194304
-    TargetPractice = 8388608
-    Key9           = 16777216
-    Coop           = 33554432
-    Key1           = 67108864
-    Key3           = 134217728
-    Key2           = 268435456
+    NoMod               = 0
```
Comment (tybug) [inline circleguard/enums.py]: inconsistent spacing (line up all the 2 letter names like the numbers are)

also all mods should be getting acronyms (might as well), use the ones from here https://github.com/christopher-dG/osu-bot/blob/master/osubot/consts.py#L103 for consistency if he has them, or make them up if he doesn't (NV for no video, K1 - K9)

Comment (InvisibleSymbol) [reply circleguard/enums.py]: Yea, will reformat it & add the missing ones (for now I just did the ones on the osu! wiki, didn't want to make anything stupid)
Gonna make the changes tommorow.

**Comment by InvisibleSymbol** [comment]
Updated it according to the comment
I used "CN" for AutoPlay since "AP" was already used for AutoPilot and Cinema is basically the same mod as AutoPlay

File: circleguard/enums.py (hunk 1 above)
Comment (samuelhklumpers) [reply circleguard/enums.py]: does this resolve it tybug

Comment (tybug) [reply circleguard/enums.py]: It does yeah, sorry for the late review, had a lot going on yesterday.

**Pull request merged**